vmdb/lib/workers/event_catcher_openstack_infra.rb

```diff
@@ -0,0 +1,80 @@
+require 'workers/event_catcher'
+
+class EventCatcherOpenstackInfra < EventCatcher
```

Comment:
Since this is strictly a copy-paste, I'd just say inherit from `EventCatcherOpenstack` and leave it all unchanged.  Having the class naming separate will be good for logging.